examples/data-objects/table-document/src/document.ts

```diff
@@ -191,9 +191,12 @@ export class TableDocument extends DataObject<{ Events: ITableDocumentEvents }>
 		this.rows = await this.root.get<IFluidHandle<SharedNumberSequence>>("rows").get();
 		this.cols = await this.root.get<IFluidHandle<SharedNumberSequence>>("cols").get();
 
-		this.forwardEvent(this.cols, "op", "sequenceDelta");
-		this.forwardEvent(this.rows, "op", "sequenceDelta");
-		this.forwardEvent(this.matrix, "op", "sequenceDelta");
```

Comment:
Im trying to reconcile the description that says puredataobject does not participate in any event forwarding, and the code above - how did this work at the first place then?

Reply:
EventForwarders can optionally pass a "source" in their constructor to **automatically** participate in event forwarding - that is, events registered on the forwarder will immediately plumb through to the "source".  But PureDataObject doesn't do this.

The second way to participate in event forwarding is this explicit approach, calling forwardEvent directly.  Almost no one does this though.  The lines you highlighted above are the only case I found.

Reply:
Note - this is why I'm taking a deprecate in `main` and remove in `next` approach, just in case there's an unknown user of this functionality they have time to make similar changes to what you see here.